<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>499</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>BioStat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushBackFromStatistic">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>410</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {font: 13pt &quot;verdana&quot;; 
background-color: rgb(255, 255, 255); 
border-radius: 2px; 
border: 2px solid #9a9; 
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)} 
QPushButton:hover {background-color: rgba(231, 253, 255, 0.7); border: 2px solid #9a9;}</string>
    </property>
    <property name="text">
     <string>НАЗАД</string>
    </property>
   </widget>
   <widget class="QLabel" name="version">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>0</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Constantia&quot;;</string>
    </property>
    <property name="text">
     <string>version 1.0.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBuild">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>160</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {font: 13pt &quot;verdana&quot;; 
background-color: rgb(255, 255, 255); 
border-radius: 2px; 
border: 2px solid #9a9; 
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)} 
QPushButton:hover {background-color: rgba(231, 253, 255, 0.7); border: 2px solid #9a9;}</string>
    </property>
    <property name="text">
     <string>Построить</string>
    </property>
   </widget>
   <widget class="QComboBox" name="chooseBox">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>50</y>
      <width>111</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>verdana</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox {font: 13pt &quot;verdana&quot;; 
background-color: rgb(255, 255, 255); 
border-radius: 2px; 
border: 2px solid #9a9; 
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)} 
QPushButton:hover {background-color: rgba(231, 253, 255, 0.7); border: 2px solid #9a9;}</string>
    </property>
    <item>
     <property name="text">
      <string>Белки</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Жиры</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Углеводы</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Ккал</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>100</y>
      <width>181</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="proteins">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="fats">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="carbohydrates">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="Kkal">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>211</width>
      <height>83</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>В среднем вы съели:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>60</y>
      <width>195</width>
      <height>301</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 13pt &quot;verdana&quot;; 
background-color: rgb(255, 255, 255); 
border-radius: 2px; 
border: 2px solid #9a9; 
box-shadow: 4px 4px 4px rgba(8, 8, 8, 0.5)} 
QPushButton:hover {background-color: rgba(231, 253, 255, 0.7); border: 2px solid #9a9;</string>
    </property>
   </widget>
   <zorder>widget</zorder>
   <zorder>pushBackFromStatistic</zorder>
   <zorder>version</zorder>
   <zorder>pushBuild</zorder>
   <zorder>chooseBox</zorder>
   <zorder>verticalLayoutWidget</zorder>
   <zorder>label_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
